### PR TITLE
Extend font encoding

### PR DESCRIPTION
**Pull request by Ralim**
This is working to extend font encoding to multibyte.
I probably have an edge case here somewhere 😓  but getting ball rolling

Refer to: https://github.com/Ralim/IronOS/discussions/847

This solves the first pressing issue: > 255 glyphs
Also ensures that CI will fail if we run out of room (oops)
@alvinhochun

### CODE REVIEW

File: Translations/make_translation.py [hunk 1]
```diff
@@ -242,14 +247,24 @@ def getFontMapAndTable(textList):
     totalSymbolCount = len(set(textList) | set(forcedFirstSymbols))
     # \x00 is for NULL termination and \x01 is for newline, so the maximum
     # number of symbols allowed with 8 bits is `256 - 2`.
-    if totalSymbolCount > (256 - 2):
+    if totalSymbolCount > ((0xEE * 0x0F) - 2):
```
Comment (playday3008) [inline Translations/make_translation.py:250]: Wow, from 255 to 3569, impressive

Comment (Ralim) [reply Translations/make_translation.py:250]: I mean, I don't think we can get there but also would rather not revisit at 500 🤣

**Comment by alvinhochun** [comment]
Haven't have time to properly look at this yet, but I'm thinking might want to rewrite some parts of the changes...

**Comment by Ralim** [comment]
1000% happy for changes, curious to see what your thoughts are

**Comment by alvinhochun** [comment]
These are my changes: https://github.com/Ralim/IronOS/compare/extend-font-encoding...alvinhochun:extend-font-encoding

**Pull request merged**